bodhi/server/models/models.py

```diff
@@ -1022,15 +1022,19 @@ def get_cvestring(self):
     def get_bug_karma(self, bug):
         good, bad, seen = 0, 0, set()
         for comment in reversed(self.comments):
-            if comment.user.name in seen:
-                continue
-            seen.add(comment.user.name)
-            for feedback in comment.bug_feedback:
-                if feedback.bug == bug:
-                    if feedback.karma > 0:
-                        good += 1
-                    elif feedback.karma < 0:
-                        bad += 1
+            if comment.user:
+                if (comment.user.name == u'bodhi' and
+                        ('New build' in comment.text or 'Removed build' in comment.text)):
+                    break
```

Comment:
Also, it seems like a bug to me that this method never had this check in it, so thanks for adding it! I do believe it is a separate concern from the purpose of your commit though - would you mind putting this (and a test for it) into a second commit so the commits are atomic and focused on one thing each? You can keep it in the same pull request if you prefer.